components/card/card.js

```diff
@@ -280,18 +283,20 @@ class Card extends FocusMixin(RtlMixin(LitElement)) {
 				@d2l-dropdown-close="${this._onDropdownClose}"
 				@d2l-tooltip-show="${this._onTooltipShow}"
 				@d2l-tooltip-hide="${this._onTooltipHide}">
-				<a @blur="${this._onLinkBlur}"
-					?download="${this.download}"
-					@focus="${this._onLinkFocus}"
-					href="${ifDefined(this.href ? this.href : undefined)}"
-					hreflang="${ifDefined(this.hreflang)}"
-					@mouseenter="${this._onLinkMouseEnter}"
-					@mouseleave="${this._onLinkMouseLeave}"
-					rel="${ifDefined(this.rel)}"
-					target="${ifDefined(this.target)}"
-					type="${ifDefined(this.type)}">
-					<span class="d2l-card-link-text d2l-offscreen">${this.text}</span>
-				</a>
+				${ this.href ? html`
```

Comment:
Is it possible removing the the anchor would break any existing behaviours for users/consumers? If so, should this change be flagged just in case?

Reply:
I was a little nervous about that too, but if the `href` is undefined, the anchor would not be focusable. There would be a slight style difference on hover (due to no `mouseenter` and `mouseleave` handlers), but that would probably be desirable. Nevertheless, I think it would be best to error on the side of caution and flag it.